faithcomesbyhearing/dbp
# FCBHDBP-298 add captcha to api key request form

# Description
It has added the captcha field to API Key Request Form.

## NOTE
We need to register and create a google reCAPTCHA version 2 for each environment using the next URL:
https://www.google.com/recaptcha/admin/create

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-298

## How Do I QA This
Go to path `api_key/request` and you should see the google captcha in the bottom of page.

## Screenshots
## How to register a google reCAPTCHA for a server
![screenshot-www google com-2022 02 07-22_15_31(1)](https://user-images.githubusercontent.com/73488660/152911941-f79f9d3f-acdc-470e-b6bf-8af39de12320.png)

##  google reCAPTCHA added into form
![screenshot-localhost_8080-2022 02 07-22_23_39](https://user-images.githubusercontent.com/73488660/152912142-66e385dc-db41-473d-beb8-870887684459.png)
